website/views/options/inventory/pets.jade

```diff
@@ -51,7 +51,7 @@ mixin petList(eggSource, potionSource)
         each t,k in env.Content.specialPets
           - var egg = k.split('-')[0], pot = k.split('-')[1]
           button(ng-if='user.items.pets["#{egg}-#{pot}"]', class="pet-button Pet-#{egg}-#{pot}", ng-class='{active: user.items.currentPet == "#{egg}-#{pot}"}', ng-click='choosePet("#{egg}", "#{pot}")', popover=env.t(t), popover-trigger='mouseenter', popover-placement='bottom')
-        a(target='_blank', href='http://habitica.wikia.com/wiki/Contributing_to_Habitica')
+        a(target='_blank', href=evt.t('contribWikiaLink'))
```

Comment:
This should be `env.t` not `evt.t`

I'm reviewing the rest of it too so there's likely to be other comments soon. I'll post to say when I'm finished.